src/all/coomer/build.gradle

```diff
@@ -0,0 +1,10 @@
+ext {
+    extName = 'Coomer'
+    extClass = '.Coomer'
+    themePkg = 'kemono'
+    baseUrl = 'https://coomer.su'
```

Comment:
not important but `baseUrl` shouldn't be added when themelib doesn't support deeplinks.

Reply:
It's added because it was a required field in the generator, and it doesn't hurt.

Also, if it's omitted and the theme package adds deep link some day, you'll need to go over all overrides and add it anyway.